CHANGELOG.txt

```diff
@@ -78,7 +78,9 @@ master:
  - obspy.io.segy:
    * Raise nicer error messages when header packing fails (see #2194, #2196).
  - obspy.io.seiscomp:
-   * Adding support for SC3ML 0.10 (see #2024).
+   * Adding support for SC3ML 0.10 (see #2004, #2024 and #2124).
+   * Adding support for SC3ML 0.11 (see #2124).
```

Comment:
Currently the event plugin only supports version 0.10. I did a PR (#2284) to support 0.11 but it isn't merged. The changelog is a bit misleading IMHO.